core/src/main/scala/kafka/network/SocketServer.scala

```diff
@@ -1228,7 +1228,6 @@ class ConnectionQuotas(config: KafkaConfig, time: Time) extends Logging {
 
   class ListenerConnectionQuota(lock: Object, listener: ListenerName) extends ListenerReconfigurable {
     @volatile private var _maxConnections = Int.MaxValue
-    private val listenerPropName = s"${listener.configPrefix}${KafkaConfig.MaxConnectionsProp}"
```

Comment:
@rajinisivaram Just to double-check, this can be deleted, we didn't intend to use for something, right?